taxi/forms.py

```diff
@@ -0,0 +1,47 @@
+from django import forms
+from django.contrib.auth import get_user_model
+from django.contrib.auth.forms import UserCreationForm
+from django.core.validators import RegexValidator
+
+from .models import Driver, Car
+
+
+class LicenseNumberField(forms.CharField):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.validators.append(RegexValidator(
+            regex=r"^[A-Z]{3}\d{5}$",
+            message=("Please enter a valid driver\'s license number "
+                     "consisting of 3 uppercase letters followed by 5 digits."
```

Comment:
I think it's redundant to refresh memory of someone, who has license, so we only help someone who doesn't know how license would look like so, I think, it's better just to ask for proper/valid driver's license number, nothing more)